src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -306,9 +295,8 @@ public function configureOptions(OptionsResolver $resolver)
         $resolver->setDefaults(array(
             'multiple' => false,
             'expanded' => false,
-            'choice_list' => null, // deprecated
             'choices' => array(),
-            'choices_as_values' => null,  // to be deprecated in 3.1
+            'choices_as_values' => true, // to be deprecated in 3.1
```

Comment:
this was done in order to be able to distinguish between somebody setting the option to true and somebody not touching the option at all. If somebody sets it to true manually, it should trigger a deprecation because he would get an exception when we remove the option.